### PR TITLE
Add minesweeper exercise

### PR DESCRIPTION
**Pull request by abhishalya**
This add a new exercise specified at
https://github.com/exercism/problem-specifications/tree/master/exercises/minesweeper.

### CODE REVIEW

**Comment by abhishalya** [comment]
Small question: Is there any way we could check if `check_notebook` passes or not?
Since until we add the notebook file to track, the diff will always be emitted by the script. Also, I think we should only be checking diff for `.ipynb` files only rather than checking for all in this script.

File: exercises/minesweeper/example.jl [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# function to check if the current mine represent a mine or not
```
Comment (logankilpatrick) [inline exercises/minesweeper/example.jl]:
```suggestion
# function to check if the current character represents a mine or not
```

File: exercises/minesweeper/example.jl [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+            else
+                ans_row *= '*'
+            end
+        end
+
+        # push the calculated row_string to our ans
+        push!(ans, ans_row)
+    end
+
+    ans
```
Comment (logankilpatrick) [inline exercises/minesweeper/example.jl]:
```suggestion
return ans
```
I guessing this needs to be returned though I am not certain.

Comment (abhishalya) [reply exercises/minesweeper/example.jl]: I'll add this change to the commit.

File: exercises/minesweeper/example.jl (hunk 1 above)
Comment (abhishalya) [reply exercises/minesweeper/example.jl]: Yep :sweat_smile:

File: exercises/minesweeper/example.jl (hunk 2 above)
Comment (logankilpatrick) [reply exercises/minesweeper/example.jl]: Cool, you can commit directly from the suggestion if that helps

Comment (abhishalya) [reply exercises/minesweeper/example.jl]: I'll prefer the old approach :P

File: config.json [hunk 3]
```diff
@@ -527,6 +527,24 @@
                 "math",
                 "integers"
             ]
+        },
+        {
+            "slug": "minesweeper",
+            "uuid": "9a47bffd-4dd2-48ce-b47b-753271305966",
+            "core": false,
+            "unlocked_by":"nucleotide-count",
```
Comment (SaschaMann) [inline config.json]:
```suggestion
            "unlocked_by": "pangram",
```

`nucleotide-count` already unlocks a lot of exercises and arrays haven't been introduced to the core progression path yet, so let's move it to be unlocked by pangram.

**Comment by abhishalya** [comment]
Thanks, I've included the change requested.

**Comment by SaschaMann** [comment]
Thanks!

**Pull request merged**